### ISSUE TITLE
Add Google Maps plugin field

### ISSUE THREAD

**Issue by engram-design** [verbb/formie#625]
Support [Google Maps plugin](https://plugins.craftcms.com/google-maps)

**Comment by lindseydiloreto** [comment]
I’m on board for making this a reality. Let me know if you need assistance with anything, I’m happy to help! 👍

**Comment by wmdhosting** [comment]
This will be great :)

I think it can be done as part of adress field as it posiblle to hide fields we dont need

Main point in my case is that  visitors can  drag and drop PIN  on map to specific location.. 

As sometimes locations are not on real address.

**Comment by engram-design** [comment]
I'm in two minds about this being part of the address field. What if we want to store multiple pins in a field? That's just not possible with the Address field, which already has an Address Provider integration for autocompletion of addresses, which is arguably pretty similar to maps.

If anything, It may very well be a thin wrapper around the already-excellent Google Maps plugin which handles the front-end and back-end side of things, you just need the ability to include it in a Formie form.

Will mull it over.

**Comment by wmdhosting** [comment]
OK then :)  yes that make sense..

**Comment by lindseydiloreto** [comment]
The good news is, it's very easy to build a front-end form with a Google Maps Address field...

https://plugins.doublesecretagency.com/google-maps/address-field/front-end-form/

**Comment by wmdhosting** [comment]
yes ok.  thx for info..  but FORMIE is something else :)

**Comment by lindseydiloreto** [comment]
I agree... I'm simply indicating that all of the puzzle pieces already exist. 🙂

If we start with that front-end form example (and the JS snippets to go along with it), that will make it much easier for Verbb to integrate the field with Formie. They won't have to dive through the plugin's source code to figure out how it should look on the front-end.

**Comment by svondervoort** [comment]
Any update on this?

Currently encountering this issue as we would like to map the Address field on the Google Maps field using an Element Integration.

Else we would have to programatically create an entry using one of formies Events.

**Comment by engram-design** [comment]
Added in [4.0.0-beta.5](https://github.com/verbb/formie/releases/tag/4.0.0-beta.5).
